ktor-utils/common/src/io/ktor/util/Attributes.kt

```diff
@@ -12,6 +12,8 @@ import kotlin.reflect.*
  * Specifies a key for an attribute in [Attributes]
  * @param T is a type of the value stored in the attribute
  * @param name is a name of the attribute for diagnostic purposes. Can't be blank
+ *
+ * [Report a problem](https://ktor.io/feedback?fqname=io.ktor.util.AttributeKey)
```

Comment:
This is rendered as a link in a parameter description as it is placed under `@param` tags

<img width="498" alt="image" src="https://github.com/user-attachments/assets/92fdf6af-9ed4-4def-9314-a93b027f73e8">

Reply:
@olme04, could you suggest is there any way to put it on the bottom line of the quick help?

Reply:
I don't think it's possible. In this case `[Report a problem]` should be above tags, as `tags` could be only written in the end of the KDoc block.

May be this could be done inside ktor IDEA plugin?

Reply:
fixed within generator